templates/docs/patterns/tiered-list/index.md

```diff
@@ -0,0 +1,92 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Tiered list | Patterns
+---
+
+The tiered list pattern is used to easily implement a list of paired titles and
+descriptions underneath a top-level title and description. An optional [CTA
+block](/docs/patterns/cta-block) is placed at the bottom of the content as well.
+Depending on the size and composition of your content, you can choose from a
+variety of tiered list layouts:
+
+- [50/50 on desktop w/ description](#5050-on-desktop-w-description)
```

Comment:
I find the `w/` `w/o` notation hard to read. I don't know how common it is or what are the rules of using it or not, but I feel like it's not really necessary.

It's especially confusing when combined with legitimate use of / in `50/50` or in our example titles:

<img width="749" alt="image" src="https://github.com/user-attachments/assets/2d81f0fc-17be-424c-aed4-c0bca22b6f54">

"Tiered list / 50/50 on desktop w/ description" - how do you read that?

I'm also pretty sure any screen reader will fail on that.

So, please just use "with" and "without" instead everywhere: links, headings in the docs, titles of examples, etc.